src/lib.rs

```diff
@@ -1918,6 +1932,40 @@ impl Connection {
         Ok(buf.len())
     }
 
+    /// Shuts down reading or writing from/to the specified stream.
+    ///
+    /// When the `direction` argument is set to [`Shutdown::Read`], outstanding
+    /// data in the stream's receive buffer is dropped, and no additional data
+    /// is added to it. Data received after calling this method is still
+    /// validated and ACKed but not stored, and [`stream_recv()`] will not
+    /// return it to the application.
```

Comment:
So IIUC until we add RESET_STREAM or STOP_SENDING, the library will continue to receive data, drop it and not insrease stream flow control window?

Reply:
Yes, though I just realized this also consumes connection-level flow control credits, so maybe we should give those back.